eng/source-build/source-build.proj

```diff
@@ -55,7 +55,7 @@
     </ItemGroup>
 
     <Exec
-      Command="git apply --ignore-whitespace --whitespace=nowarn &quot;%(SourceBuildPatchFile.FullPath)&quot;"
+      Command="git --work-tree=$(ProjectRoot) apply --ignore-whitespace --whitespace=nowarn &quot;%(SourceBuildPatchFile.FullPath)&quot;"
```

Comment:
I based `ProjectRoot` based on the `WorkingDirectory` set on the next line; is there a better variable name I should be using instead?

Reply:
I think `ProjectRoot` is correct here.